package.json

```diff
@@ -38,6 +38,7 @@
     "@edx/frontend-platform": "5.6.1",
     "@edx/openedx-atlas": "^0.6.0",
     "@edx/paragon": "20.46.3",
+    "@openedx/paragon": "^21.5.7",
```

Comment:
i'm not sure how this made it in but we actually don't need `@openedx/paragon` since it's not being used anywhere in the code yet. we were still on `@edx/pargon`. I removed `@openedx/paragon`